### DISCUSSION TITLE
涩图发不出相关问题请参考此处

### DISCUSSION THREAD

**Discussion by xipesoy** [Tsuk1ko/CQ-picfinder-robot#187] [一般讨论]
convert from #186

**Comment by xipesoy** [comment]
我收到涩图 只有链接和镜像地址没有图片aqa

**Comment by Tsuk1ko** [comment]
那就是发图片出了问题，如果你网络直连 i.pximg.net 不行就设置在线图片反代

**Comment by xipesoy** [comment]
没有国外服务器只有国内的能整吗aqa

**Comment by Tsuk1ko** [comment]
- 方法1：设置 [`bot.setu.pximgProxy`](https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/%E9%99%84%E5%8A%A0%E5%8A%9F%E8%83%BD#pximgproxy%E8%AF%B4%E6%98%8E)
- 方法2：设置 `bot.proxy` 让本地 pximg 反代走这个代理

**Comment by xipesoy** [comment]
aqa 现在能发图了 不知道是速度慢还是啥… 五张只会发两张左右的图片 要等几分钟 有没有啥解决办法呀owo

**Comment by Tsuk1ko** [comment]
设置 `bot.setu.size1200` 为 `true` 来发送压缩过的图，网络慢也没其他办法咯
